chart/files/pod-template-file.kubernetes-helm-yaml

```diff
@@ -19,6 +19,13 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: dummy-name
+  labels:
+    tier: airflow
+    component: worker
+    release: {{ .Release.Name }}
```

Comment:
I didn't set the `chart` label, however, as it felt odd given the scheduler is creating these. I'm happy to add it if others think we should.